pkgs/development/tools/misc/gede/default.nix

```diff
@@ -1,17 +1,17 @@
-{ stdenv, fetchurl, makeWrapper, python, qmake, ctags, gdb }:
+{ stdenv, fetchurl, makeWrapper, python, qmake, ctags, gdb, wrapQtAppsHook }:
```

Comment:
Instead of using the `wrapQtAppsHook` would using QT's `mkDerivation` work better here?

Reply:
Yes. Fixed. Thanks.